.travis.yml

```diff
@@ -7,6 +7,9 @@ branches:
   only:
   - master
 
+env:
+- DBUS_SESSION_BUS_ADDRESS=/dev/null
```

Comment:
adding this based on: https://github.com/angular/protractor/issues/2419#issuecomment-213112857